external-crates/move/crates/move-cli/src/sandbox/commands/test.rs

```diff
@@ -428,31 +428,16 @@ fn add_update_baseline_fix(s: impl AsRef<str>) -> String {
 }
 
 fn format_diff(expected: impl AsRef<str>, actual: impl AsRef<str>) -> String {
-    use difference::*;
-
-    let changeset = Changeset::new(expected.as_ref(), actual.as_ref(), "\n");
-
-    let mut ret = String::new();
-
-    for seq in changeset.diffs {
-        match &seq {
-            Difference::Same(x) => {
-                ret.push_str(x);
-                ret.push('\n');
-            }
-            Difference::Add(x) => {
-                ret.push_str("\x1B[92m");
-                ret.push_str(x);
-                ret.push_str("\x1B[0m");
-                ret.push('\n');
-            }
-            Difference::Rem(x) => {
-                ret.push_str("\x1B[91m");
-                ret.push_str(x);
-                ret.push_str("\x1B[0m");
-                ret.push('\n');
-            }
-        }
-    }
-    ret
+    use colored::Colorize;
+    use similar::ChangeTag::*;
```

Comment:
Nit, we would probably prefer as we have been bitten once too many times around the variant actually being treated as a local variable. 
```suggestion
    use similar::ChangeTag;
```

Reply:
Done :)